styles.css

```diff
@@ -1,6 +1,6 @@
 @media (max-width: 550px) {
     html {
-      font-size: 45%;
+        font-size: 45%;
```

Comment:
The CSS file works perfectly, but for readability reasons, it is better to use 2 spaces instead of TAB as a good practice. for more information, visit [HTML & CSS best practices](https://github.com/microverseinc/curriculum-html-css/blob/main/articles/html_css_best_practices.md)